modules/admin/admin-impl/src/main/java/com/enonic/xp/admin/impl/json/content/ResolveForDeleteJson.java

```diff
@@ -0,0 +1,33 @@
+package com.enonic.xp.admin.impl.json.content;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import com.enonic.xp.content.ContentIds;
+
+public class ResolveForDeleteJson
+{
+    private final ContentIds toDelete;
+
+    private final ContentIds toUnpublish;
+
+    public ResolveForDeleteJson( final ContentIds toDelete, final ContentIds toUnpublish )
```

Comment:
I prefer to use `List<ContentId>` or `List<ContentIdJson>` instead of `ContentIds`